MinPlatformPkg/Test/Library/TestPointCheckLib/DxeTestPointCheckLib.inf

```diff
@@ -35,10 +35,11 @@
   PciSegmentInfoLib
   SafeIntLib
   PlatformConfigCheckLib # MU_CHANGE: Support platform level configuration testing
+  TestPointPciSpeedInfoLib
 
 [Packages]
-  MinPlatformPkg/MinPlatformPkg.dec
   MdePkg/MdePkg.dec
+  MinPlatformPkg/MinPlatformPkg.dec
   MdeModulePkg/MdeModulePkg.dec
   IntelSiliconPkg/IntelSiliconPkg.dec
```

Comment:
Since you already moved `MdePkg.dec`, can you just go ahead and move the others in order of package stability:

```suggestion
  MdePkg/MdePkg.dec
  MdeModulePkg/MdeModulePkg.dec
  IntelSiliconPkg/IntelSiliconPkg.dec
  MinPlatformPkg/MinPlatformPkg.dec
```